testing/perl-text-unidecode/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Automatically generated by apkbuild-cpan, template 2
+# Contributor: Timothy Legge <timlegge@gmail.com>
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+pkgname=perl-text-unidecode
+_pkgreal=Text-Unidecode
+pkgver=1.30
+pkgrel=0
+pkgdesc="Provide plain ASCII transliterations of Unicode text"
+url="https://metacpan.org/release/Text-Unidecode/"
+arch="noarch"
+license="GPL-1.0-or-later Artistic-1.0-Perl"
```

Comment:
use 'OR' SPDX identifier to indicate the choice is between either of those licenses.

Reply:
@maxice8 I made the change.  Wondered about that

Reply:
Thank you

Reply:
also fixed apkbuild-cpan.in PR tto abuild